.github/styles/vocab.txt

```diff
@@ -105,6 +105,7 @@ Firekube
 Firestore
 Fiverr
 Francesco
+gerrit
```

Comment:
The "api-report" check failed.

Reply:
I see. Seems like `Bitbucket` was changed to `bitbucket`, too, when the new catalog-backend plugin for it as introduced.

I guess the api-report generator would need to be adjusted to put the package name into code formatting instead of just quotes in order to get them ignored.

Probably, you wouldn't need both entries I guess as there is no enforcement of upper case for the first letter anymore.